seed/tests/test_api.py

```diff
@@ -171,7 +169,7 @@ def test_organization(self):
         self.assertEqual(r['organizations'][0]['owners'][0]['first_name'], 'Jaqen')
         self.assertEqual(r['organizations'][0]['cycles'], [
             {
-                'name': str(date.today().year - 1) + ' Calendar Year',
```

Comment:
I think we want deletions vs the additions here.

Reply:
I think you're right, I'll change that